tests/test_exception_handling.py

```diff
@@ -1,12 +1,16 @@
+from http import HTTPStatus
+
 import pytest
 from rest_framework.test import APIClient
 
+from winter.exceptions import WinterException
 from .controllers.controller_with_exceptions import CustomException
 from .controllers.controller_with_exceptions import ExceptionWithoutHandler
 from .entities import AuthorizedUser
 
 
 @pytest.mark.parametrize(['url_path', 'expected_status', 'expected_body'], (
+        ('redirect', HTTPStatus.FOUND, '/root/'),
```

Comment:
Remove this or `test_controller_with_redirect_exception.py`

Reply:
Done